Shopping List/ShoppingListCollectionViewController.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  ShoppingListCollectionViewController.swift
+//  Shopping List
+//
+//  Created by Alex on 5/3/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
```

Comment:
This is also something that you should clean up, but on a side note, having this private constant outside your class puts it in the module namespace rather than the class namespace. We will learn about this in more detail, but basically if it is private outside of the class it is accessible by the entire module as ```reuseIdentifier```, whereas a private class constant would be available inside it's class, and not available to any other class in the module.  In most cases, its best to have it scoped to class.